latch_cli/services/move.py

```diff
@@ -45,14 +45,21 @@ def move(
 
     multi_src = len(srcs) > 1
 
-    if multi_src and dest_data.is_parent:
+    if not (dest_data.exists() or dest_data.is_direct_parent()):
+        click.echo(get_path_error(dest, "not found", acc_id))
+        raise click.exceptions.Exit(1)
+    elif multi_src and not dest_data.exists():
         click.secho(
             f"Remote destination {dest} does not exist. Cannot move multiple source"
             " files to a destination that does not exist.",
             fg="red",
         )
         raise click.exceptions.Exit(1)
-    elif multi_src and dest_data.type in {LDataNodeType.obj, LDataNodeType.link}:
+    elif (
+        multi_src
+        and (dest_data.exists() or dest_data.is_direct_parent())
+        and dest_data.type in {LDataNodeType.obj, LDataNodeType.link}
```

Comment:
```suggestion
        multi_src
        and dest_data.type in {LDataNodeType.obj, LDataNodeType.link}
```
reasoning is that if any parent is an obj/link, the whole command should fail